cookbook/templates/default/singularity-init.erb

```diff
@@ -0,0 +1,13 @@
+#!upstart
+description "Singularity Service"
+
+env PATH=/usr/local/singularity/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/lib64/qt-3.3/bin:/usr/local/bin:/bin:/usr/bin
+env MESOS_HOME=/usr/local
+env MESOS_NATIVE_LIBRARY=/usr/local/lib/libmesos.so
```

Comment:
9 times out of 10 this is accurate, but it'd be good to make customizable just in case